src/components/Metrics.js

```diff
@@ -1,9 +1,17 @@
-import React from 'react'
-import { Box, GU, textStyle, useLayout, useTheme } from '@1hive/1hive-ui'
+import React, { useState, useMemo, useCallback } from 'react'
+import {
+  Box,
+  DropDown,
+  GU,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@1hive/1hive-ui'
 
 import { useAppState } from '../providers/AppState'
 import { useUniswapHnyPrice } from '../hooks/useUniswapHNYPrice'
 import { formatDecimals, formatTokenAmount } from '../utils/token-utils'
+import { useCurrencies } from '../hooks/useCurrencies'
```

Comment:
One more small nitpick.

We generally order imports as well in the following order: 

 - External libraries
- Components
- Hooks
- Libraries
- Assets
```suggestion
import { useAppState } from '../providers/AppState'
import { useCurrencies } from '../hooks/useCurrencies'
import { useUniswapHnyPrice } from '../hooks/useUniswapHNYPrice'
import { formatDecimals, formatTokenAmount } from '../utils/token-utils'
```